### ISSUE TITLE
Sweetalert console error

### ISSUE THREAD

**Issue by chelaruc** [creativetimofficial/material-dashboard-pro-angular#126]
In `src/app/components/sweetalert/sweetalert.component.ts` we need to replace the lines 98-102:
````
swal({  title: 'Auto close alert!',
                    text: 'I will close in 2 seconds.',
                    timer: 2000,
                    showConfirmButton: false
                });
````
with
````
swal({  title: 'Auto close alert!',
                    text: 'I will close in 2 seconds.',
                    timer: 2000,
                    showConfirmButton: false
                }).then(
                    function () {},
                    // handling the promise rejection
                    function (dismiss) {
                    if (dismiss === 'timer') {
                        console.log('I was closed by the timer')
                    }
                });
````
This will be fixed in v1.5.0!

**Comment by osnersanchez** [comment]
Good day, I have seen this problem, but this is not the only problem that you had in the alerts, when you click on the swal container to close the alert in the console it shows an error that refers to overlay, this happens once the page for the first time They would have to implement the Handling Dismissals

**Comment by chelaruc** [comment]
@osnersanchez You need to add this: `.catch(swal.noop)`. This will be fixed in v1.5.0!
You can see here full answer: [https://github.com/limonte/sweetalert2#handling-dismissals](https://github.com/limonte/sweetalert2#handling-dismissals).

**Comment by osnersanchez** [comment]
thank you very much for your attention, this error also appears when the cancel button is used in any alert, the error is related to the dismissive vicable, never enters function (dismiss) {...} 

(by google translator)

**Comment by chelaruc** [comment]
fixed in v1.4.6
